client/src/main/java/tds/assessment/Segment.java

```diff
@@ -24,12 +24,17 @@
     private int fieldTestMaxItems;
     private Instant fieldTestStartDate;
     private Instant fieldTestEndDate;
+
+    // Fixed-form specific fields
     private List<Form> forms;
-    private List<Item> items;
+
+    // Adaptive-specific fields
     private Set<Strand> strands;
+    private List<Item> items;
```

Comment:
Small thing, but I read this as saying that the Items are only specific to the Adaptive algorithm.  Maybe moving Items above or just putting an extra line break.